docs/src/live/config/withStyledDraggables.tsx

```diff
@@ -62,73 +64,59 @@ export const withStyledDraggables = (components: any) => {
     {
       key: ELEMENT_H1,
       styles: {
-        gutterLeft: {
-          padding: '2em 0 4px',
-          fontSize: '1.875em',
-        },
-        blockToolbarWrapper: {
-          height: '1.3em',
-        },
+        gutterLeft: css`
+          padding: 2em 0 4px;
```

Comment:
not that it matters now, but maybe we want that 4px to be 0.25em for consistency?